### PR TITLE
Update jit-optimization-and-debugging.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Helps fix https://github.com/dotnet/AspNetCore.Docs/issues/4155

### CODE REVIEW

**Comment by PRMerger8** [comment]
@Rick-Anderson : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/debugger/jit-optimization-and-debugging.md [hunk 1]
```diff
@@ -52,15 +52,40 @@ There are two situations where turning on this option will **NOT** work:
 1. In situations where you are attaching the debugger to an already running process, this option will have no effect on modules that were already loaded at the time the debugger was attached.
 2. This option has no effect on DLLs that have been pre-compiled (a.k.a ngen'ed) to native code. However, you can disable usage of pre-compiled code by starting the process with the environment variable **'COMPlus_ReadyToRun'** set to **'0'**. This will tell the .NET Core runtime to disable the use of pre-compiled images, forcing the runtime to JIT compile framework code. 
 
-    > [!IMPORTANT]
-    > If you are targeting .NET Framework or an older version of .NET Core (2.x or lower), also add the environment variable 'COMPlus_ZapDisable' and set it to '1'
+   If you are targeting .NET Framework, add the environment variable 'COMPlus_ZapDisable' and set it to '1' by adding it to each profile in the *Properties\launchSettings.json* file:
```
Comment (Rick-Anderson) [inline docs/debugger/jit-optimization-and-debugging.md:55]:
```suggestion
   If you are targeting .NET Framework, add the environment variable 'COMPlus_ZapDisable' and set it to '1' 
   
Set `"COMPlus_ReadyToRun": "0"`  by adding it to each profile in the *Properties\launchSettings.json* file:
```

**Comment by PRMerger12** [comment]
@Rick-Anderson : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

**Review by Mikejo5000 — APPROVED** [review]

**Comment by Mikejo5000** [comment]
#sign-off Thanks, Rick!

**Comment by PRMerger5** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**